Runtime/Triangulator.cs

```diff
@@ -1194,15 +1036,15 @@ private bool ValidatePair(int i, int j)
         [BurstCompile]
         private struct ConstrainEdgesJob : IJob
         {
-            private NativeReference<Status> status;
+            public NativeReference<Status> status;
             [ReadOnly]
-            private NativeArray<float2> outputPositions;
-            private NativeArray<int> triangles;
+            public NativeArray<float2> positions;
+            public NativeArray<int> triangles;
             [ReadOnly]
-            private NativeArray<int> inputConstraintEdges;
-            private NativeList<Edge> internalConstraints;
-            private NativeArray<int> halfedges;
-            private readonly int maxIters;
+            public NativeArray<int> inputConstraintEdges;
+            public NativeList<Edge> internalConstraints;
+            public NativeArray<int> halfedges;
+            public int maxIters;
```

Comment:
You can preserve the original job constructors (I prefer that way), by passing the job struct, e.g.

```csharp
[BurstCompile]
private struct ConstrainEdgesJob : IJob
{
   private NativeReference<Status> status;
   ...
   public ConstrainEdgesJob(TriangulationJob job, ...) // <- probably halfedges etc here
   { 
              
   }
```

then

```csharp
   new ConstrainEdgesJob(this, halfedges, ...).Execute();
```

This will reduce changes in the PR. 
In my opinion, this makes the job construction more readable and improves the overall flow in `TriangulationJob.Execute()`.

Reply:
In my opinion this increases coupling significantly, and makes it harder to re-use these methods (job structs) for anything else. But I'll make this change where it makes sense.

Reply:
Thanks! Regarding coupling, we can always add constructors to make jobs more general, but for now, they are private. I believe that after your refactor, there will be room for further cleanup, which I hope will eliminate these 'internal' jobs entirely.

Reply:
After we change the jobs to methods, we can no longer just add additional constructors.

Reply:
Oh, I noticed that you made a change based on my suggestion. However, I believe there might have been a misunderstanding. What I meant by my comment was not to have a mix of passing some job data in the constructor and some in the initialization list. If it's possible, we could revert to the previous solution where all data was initialized in the initialization list for consistency. 

My initial suggestion may not be as important.

Reply:
I added it to more jobs now. Personally, I just think it adds more unnecessary lines of code. But it's your call.
Note that I cannot just take a single struct as the input, because most inputs now come from local variables, not from the surrounding struct. I also think the named parameters that I use now are necessary for clarity.

Reply:
> Note that I cannot just take a single struct as the input, 

Ok! I haven't checked this properly. It seems the previous proposal, which involved using initialization lists, might have been better. If it's not too much trouble, we could revert to it, or alternatively, I could address this later in a separate pull request.

Reply:
done